web-common/src/features/dashboards/time-series/MetricsTimeSeriesCharts.svelte

```diff
@@ -182,25 +183,12 @@
     $timeControlsStore.selectedTimeRange &&
     !isScrubbing
   ) {
-    if (!mouseoverValue?.x || !(mouseoverValue.x instanceof Date)) {
-      chartInteractionColumn.update((state) => ({
-        ...state,
-        hover: undefined,
-      }));
-    } else {
-      const { position: columnNum } = bisectData(
-        mouseoverValue.x,
-        "center",
-        "ts_position",
-        isAllTime ? formattedData?.slice(1) : formattedData?.slice(1, -1),
-      );
-
-      if ($chartInteractionColumn?.hover !== columnNum)
-        chartInteractionColumn.update((state) => ({
-          ...state,
-          hover: columnNum,
-        }));
-    }
+    updateChartInteractionStore(
+      mouseoverValue?.x,
+      undefined,
+      isAllTime,
+      formattedData,
+    );
```

Comment:
If possible, it'd be nice if this could be moved to `<MeasureChart on:chart-hover ... />` to mirror `<TDDAlternateChart on:chart-hover ... />`, which is clean.

Reply:
That would require some refactor. Mouse events are stored reactively and don't have a direct dispatch even available. Probably something to look into when focusing on tech debt.